masterlist.yaml

```diff
@@ -1718,6 +1718,27 @@ plugins:
       - <<: *obsolete
         subs: [ '[Better Game Performance v5](https://www.nexusmods.com/newvegas/mods/51871/)' ]
 
+  - name: 'SD_Capture( TTW Additions)?\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/newvegas/mods/82597'
```

Comment:
Please add a trailing slash `/` at the end of the link.

Reply:
Your latest commit added a `é` instead of a trailing slash `/` at the end of the url.

Reply:
Canadian french keyboard strikes again. I'll fix it later tonight, not to worry